stakepooldclient/stakepooldclient.go

```diff
@@ -498,3 +498,65 @@ func (s *StakepooldManager) RPCStatus() []string {
 
 	return stakepooldPageInfo
 }
+
+// comaprePubkeys performs fn on all key, value pairs and returns
+// whether any are true
+func (s *StakepooldManager) comparePubKeys(fn func(string, string) bool) (bool, error) {
+	for i, conn := range s.grpcConnections {
+		client := pb.NewStakepooldServiceClient(conn)
+		resp, err := client.MyMasterPubKeys(context.Background(), &pb.MyMasterPubKeysRequest{})
+		if err != nil {
+			return false, fmt.Errorf("GetMasterPubkeys gRPC failed on stakepoold instance %d: %v", i, err)
+		}
+		for _, v := range resp.MasterPubKeys {
+			if fn(v.Account, v.PubKey) {
+				return true, nil
+			}
+		}
+	}
+	return false, nil
+}
+
+// WalletsHavePubKey checks whether pubkey exists on any stakepoold
+// wallets
+func (s *StakepooldManager) WalletsHavePubKey(pubkey string) (bool, error) {
+	hasPubkeyFn := func(k string, v string) bool {
+		return v == pubkey
+	}
+	hasPubKey, err := s.comparePubKeys(hasPubkeyFn)
+	if err != nil {
+		return false, err
+	}
+	return hasPubKey, nil
+}
+
+// DefaultAccountPubKey gets the master extended public key for the
+// default account. Returns an error if all pubkeys are not the same.
```

Comment:
The same as what? Need to mention this is comparing these pubkeys between each wallet.